ipa-core/src/telemetry/mod.rs

```diff
@@ -33,84 +31,20 @@ pub mod metrics {
             }
         }
 
-        impl From<RequestProtocolVersion> for KeyName {
+        impl From<RequestProtocolVersion> for &'static str {
```

Comment:
Maybe this should be an `as_str` method?

Reply:
its a little less ergonomic at the callsite, but not too bad